<launch>
    <arg name="sim" default="false"/>

    <node pkg="bitbots_ball_filter" exec="robot_filter" name="bitbots_ball_filter" output="screen">
        <param from="$(find-pkg-share bitbots_ball_filter)/config/params.yaml"/>
        <param name="odom_frame" value="odom"/>
        <param name="map_frame" value="map"/>
        <param name="use_sim_time" value="$(var sim)"/>
    </node>
</launch>
